massgov/mayflower
# DP-19233 Correct dataviz sizes

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Correct the width of the small size  `<figure>` to 33% of its parent container width, and the medium size to 50%.
Clean up unused styles for `<figure>`.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-19233)
- [Github issue]()

## Steps to Test


Final spec. for the figure width
| Figure size               | < 320px | 321px - 480px | 481px - 620px   | 612px - 780px | 781px - 910px | 911px - 1014px | 1025px < |
|--------------------|:--------:|:---------------:|:----------------:|:--------------:|:--------------:|:---------------:|:---------------:|
| Small                        | 100%      | 100%                | ~~50%~~ 100% | ~~50%~~  ~~1/3~~ 50%  | 1/3 | 1/3 | 1/3 |
| Small + right col.     | 100%      | 100%                | 100%                   |  100%               | 100%             | 100% | 50% |
| Medium                    | 100%      | 100%                | ~~2/3~~ 100%   | ~~2/3~~ 50%  | 50%                 | 50% | 50% |
| Medium + right col. | 100%      | 100%                | 100%                   | 100%                | 100%             | 100% | 50% |
| Large                        | 100%      | 100%                | ~~2/3~~ 100%   | ~~2/3~~ 100% | ~~50%~~ 100% | 2/3 | 2/3 |
| Large + right col.     | 100%      | 100%                | 100%   | 100% | 100% | 100% | 100% |

Since testing requires adding multiple tableau samples, it'd be easier to test in Drupal.

1. Build the site with this branch in Drupal.

2. Use an information detail page, add one set of 7 tableau contents with small and one set with medium sizes.  Both sets should have text wrapping.  You can choose either left or right for alignment.  These samples need to be added together in one section for one size.  One section has 7 small tableau items. Another section has 7 tableau items.  Or, you could add 14 tableau items, 7 smalls and 7 mediums.  No other content should be in between them in a set.  You can add other content like a paragraph before and after a set.

3. Add 1 or 2 large size tableau content.

3. Save the edit and preview the page.

3.  Check the `<figure>` width and its parent container `<div>` width through the inspect in the browser.

3.  Find the ratios of the `<figure>` width against the parent container `<div>` meet the spec. above in each breakpoint.

## Screenshots

Small in 1 column
<img width="751" alt="small-1col_621" src="https://user-images.githubusercontent.com/9633303/99598511-a34e9900-29c7-11eb-9e78-3dd3547bd7a5.png">

<img width="836" alt="small-1col_781" src="https://user-images.githubusercontent.com/9633303/99598525-a8abe380-29c7-11eb-8cf1-27bf5c891c34.png">

Small with right column
<img width="757" alt="small-right-col_911" src="https://user-images.githubusercontent.com/9633303/99598557-b6f9ff80-29c7-11eb-89db-c07b4d1475d8.png">

<img width="831" alt="small-right-col_1025" src="https://user-images.githubusercontent.com/9633303/99598571-bb261d00-29c7-11eb-89f6-151841fd8292.png">

Medium in 1 column
<img width="715" alt="medium-1col_621" src="https://user-images.githubusercontent.com/9633303/99598596-c5481b80-29c7-11eb-9664-14fe11dbb12c.png">

<img width="832" alt="medium-1col_781" src="https://user-images.githubusercontent.com/9633303/99598608-c8dba280-29c7-11eb-966a-d3a312e762a3.png">

Medium with right column
<img width="741" alt="medium-right-col_911" src="https://user-images.githubusercontent.com/9633303/99598631-d3963780-29c7-11eb-89d3-12449c9d0720.png">

<img width="836" alt="medium-right-col_1025" src="https://user-images.githubusercontent.com/9633303/99598643-d85aeb80-29c7-11eb-8eb3-afce6386b617.png">

Large in 1 column
<img width="828" alt="large-1ccol_781" src="https://user-images.githubusercontent.com/9633303/99598680-ea3c8e80-29c7-11eb-9a6d-b36cbee19abc.png">

<img width="744" alt="large-1col_911" src="https://user-images.githubusercontent.com/9633303/99598692-ed377f00-29c7-11eb-9de7-22066c8b08e1.png">

Large with right column
<img width="776" alt="large-right-col_911" src="https://user-images.githubusercontent.com/9633303/99598709-f58fba00-29c7-11eb-84ea-1027de94ad50.png">

<img width="828" alt="large-right-col_1025" src="https://user-images.githubusercontent.com/9633303/99598714-f88aaa80-29c7-11eb-9996-718f9d066117.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
